core/domain/action_registry_test.py

```diff
@@ -17,27 +15,87 @@
 """Tests for methods in the action registry."""
 
 from __future__ import annotations
-
+from unittest import mock
 from core.domain import action_registry
 from core.tests import test_utils
+from typing import Any, Dict
 
 
 class ActionRegistryUnitTests(test_utils.GenericTestBase):
     """Test for the action registry."""
 
+    @mock.patch('core.domain.action_registry.Registry.get_all_action_types')
+    @mock.patch('importlib.import_module')
+    def test_ancestors_name(
```

Comment:
Can you please update the test name, according to this format:
``test_{{action}}_with_{{with_condition}}_{{has_expected_outcome}}``